packages/samples/atomic/index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Atomic Components</title>
+
+    <script type="module" src="https://unpkg.com/@coveo/atomic@0.7.0-alpha.33/dist/atomic/atomic.esm.js"></script>
+    <link rel="stylesheet" href="https://unpkg.com/@coveo/atomic@0.7.0-alpha.33/dist/atomic/themes/default.css" />
+    <script>
+      (async () => {
+        await customElements.whenDefined('atomic-search-interface');
+        const searchInterface = document.querySelector('atomic-search-interface');
+
+        // Initialization
+        await searchInterface.initialize({
+          accessToken: 'xx564559b1-0045-48e1-953c-3addd1ee4457',
+          organizationId: 'searchuisamples',
+        });
+
+        // Trigger a first search
+        searchInterface.executeFirstSearch();
+
+        // Add/modify a translation
+        // searchInterface.i18n.addResource('en', 'translation', 'search', 'Make me feel lucky!');
+        // Change the language of the interface dynamically
+        // searchInterface.language = 'fr';
+        // Interact with the engine
+        // searchInterface.engine.dispatch(...);
+        // Get all strings for a language/namespace
+        // const allStrings = searchInterface.i18n.getResourceBundle('en', 'translation');
+      })();
+    </script>
+    <style>
+      /* Example of CSS variables customiætion */
+      :root {
+        --atomic-primary: blue;
+        --atomic-secondary: blue;
+        --atomic-separator: ' - ';
+      }
+
+      /* Example of result templates CSS parts customization */
+      /* atomic-result::part(result-link):hover {
+        color: orange;
+      } */
+    </style>
+    <style>
+      /* Example of generic layout styling  */
+      .interface-container {
+        width: 100%;
+        max-width: 1250px;
+        margin: 0 auto;
+        display: grid;
+        grid-template-columns: 25% 1fr;
+        gap: 20px;
+      }
+
+      .search-box-item {
+        grid-column-start: 2;
+      }
+
+      .footer-item {
+        grid-column-start: 2;
+        grid-row-start: 3;
+      }
+
+      .side-by-side-container {
+        display: flex;
+        justify-content: space-between;
+        align-items: flex-end;
+      }
+
+      atomic-facet-manager {
+        padding-top: 13px;
+      }
+
+      atomic-result-list {
+        display: block;
+        margin-top: 10px;
+      }
+
+      atomic-result::part(result-printable-uri-list-element) {
+        text-transform: capitalize;
+      }
+    </style>
+  </head>
+  <body>
+    <atomic-search-interface id="search">
+      <div class="interface-container">
```

Comment:
For SEO, perhaps we could use/recommend semantic elements?

For instance, this div (interface-container) could be a `main` (if we expect it to be the most important part of the webpage) or a `section` with an `id` instead of a class.